app/model/commands/ActiveAssetCommand.scala

```diff
@@ -49,4 +57,43 @@ case class ActiveAssetCommand(
       AssetVersionConflict
     }
   }
+
+  private [commands] def firstFrameImageName(mp4Name: String): String = {
```

Comment:
TIL `[package]` = _qualified_ private

private to the `commands` package rather than just the `ActiveAssetCommand` class 👍